docs/pipelines.md

```diff
@@ -26,6 +26,7 @@ weight: 203
         - [Compose using Pipelines in Pipelines](#compose-using-pipelines-in-pipelines)
       - [Guarding a `Task` only](#guarding-a-task-only)
     - [Configuring the failure timeout](#configuring-the-failure-timeout)
+	- [Pipelines in Pipelines](#pipelines-in-pipelines)
```

Comment:
Suggest renaming to:

```
- [Specifying `Pipelines` in PipelineTasks](#Specifying `Pipelines` in PipelineTasks)
```

And moving it to one section below https://github.com/tektoncd/pipeline/blob/05b1078f0a0db12b1356c5b5b92359d436f8ffe1/docs/pipelines.md#specifying-remote-tasks

Reply:
Thanks @khrm. This feature is generally known as pipeline in pipeline. The idea here for this suggestion is to align with the rest of the specification in this [doc](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#pipelines). I am open to renaming the doc `pipelines-in-pipelines.md` if needed.

Reply:
Thanks @khrm I have deleted this file for now.